hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordPayload.java

```diff
@@ -44,6 +44,11 @@
   @PublicAPIMethod(maturity = ApiMaturityLevel.DEPRECATED)
   T preCombine(T oldValue);
 
+  @PublicAPIMethod(maturity = ApiMaturityLevel.DEPRECATED)
```

Comment:
Let's add javadoc here. It should be clear to the users why we need this API with schema